Makefile.dune

```diff
@@ -34,6 +37,21 @@ world: voboot
 watch: voboot
 	dune build $(DUNEOPT) @install -w
 
+check: voboot
+	dune build $(DUNEOPT) @check
+
+COQTOP_FILES=ide/idetop.bc ide/coqide_main.bc checker/coqchk.bc
+PLUGIN_FILES=$(wildcard plugins/*/*.mlpack)
+PRINTER_FILES=dev/top_printers.cma dev/checker_printers.cma
+QUICKBYTE_TARGETS=$(COQTOP_FILES) $(PLUGIN_FILES:.mlpack=.cma) $(PRINTER_FILES) topbin/coqtop_byte_bin.bc
+QUICKOPT_TARGETS=$(COQTOP_FILES:.bc=.exe) $(PLUGIN_FILES:.mlpack=.cmxa) $(PRINTER_FILES:.cma=.cmxa) topbin/coqtop_bin.exe
```

Comment:
I'm not so sure that PRINTER_FILES in opt are of any use. They are typically loaded the the ocaml toplevel for getting better `#trace` display.

Reply:
They are included in `ocamlopt` which is what this target emulates. Maybe I should call it this way? Anyways, it is important to compile them and they take very little time.